ofxclient/config.py

```diff
@@ -116,6 +118,11 @@ def secure_items(self, section):
     def set(self, section, option, value):
         """Set an option value. Knows how to set options properly marked
         as secure."""
+        print("set [%s] %s to: %s (%s)" % (
+            section, option, value, type(value)
+        ))
```

Comment:
This print statement needs to be removed or converted into a `logging` module call